tests/foreman/ui/test_http_proxy.py

```diff
@@ -194,3 +194,50 @@ def test_positive_assign_http_proxy_to_products_repositories(session, module_org
         assert repo_b2_values['repo_content']['http_proxy_policy'] == proxy_policy.format(
             http_proxy_b.name
         )
+
+
+@pytest.mark.tier1
+@pytest.mark.run_in_one_thread
+@pytest.mark.destructive
```

Comment:
Why using destructive mark here? confirm if this is intended?

Reply:
Yes, it will affect other tests as the settings get changed and mostly the tests based on syncing of the repository.

Reply:
I don't think you need this mark though. The `setting_update` fixture will restore the original value for `content_default_http_proxy` after your test case runs (pass or fail). 

For future use, when you do have a destructive test, that test should also use `destructive_sat` as a dependent fixture, so that a fresh satellite is deployed for the scope of that test case.  This should be done whenever the destructive test mark is used.